CommonTools/RecoAlgos/interface/RecoTrackSelectorBase.h

```diff
@@ -19,6 +19,8 @@ class RecoTrackSelectorBase {
     ptMin_(cfg.getParameter<double>("ptMin")),
     minRapidity_(cfg.getParameter<double>("minRapidity")),
     maxRapidity_(cfg.getParameter<double>("maxRapidity")),
+    meanPhi_((cfg.getParameter<double>("minPhi")+cfg.getParameter<double>("maxPhi"))/2.),
+    rangePhi_((cfg.getParameter<double>("maxPhi")-cfg.getParameter<double>("minPhi"))/2.),
```

Comment:
One possibility would be to configure directly with `meanPhi` and `rangePhi` (or maybe call it `deltaPhi`).